Spread/AdminSpread.php

```diff
@@ -35,12 +34,13 @@ class AdminSpread implements SpreadInterface
     protected $userClass;
 
     /**
-     * @param UserManagerInterface $userManager
+     * @param RegistryInterface $registry
+     * @param string            $userClass
      */
-    public function __construct(RegistryInterface $registry, $userManager)
+    public function __construct(RegistryInterface $registry, $userClass)
     {
-        $this->registry = $registry;
-        $this->userClass = $userManager;
+        $this->registry  = $registry;
```

Comment:
It's mandatory for arrays.

For assignation, we deactivates rules because could be useful on some cases, sometime not.

Already discussed with @rande.